src/firefly/js/visualize/WorkspaceCntlr.js

```diff
@@ -258,6 +258,26 @@ export function getWorkspaceStatus() {
     return isEmpty(getWorkspaceList()) && get(flux.getState(), [WORKSPACE_PATH, 'status']);
 }
 
+export function getWorkspaceErrorMsg(){
+    const statusMsg = getWorkspaceStatus();
+    //The status code and corresponding messages are in ResponseMessage.java
+    var errorMsg;
+    switch (statusMsg){
+        case 'Unauthorized':
+            errorMsg = 'You have not logged in';
+            break;
```

Comment:
From the ticket IRSA-1598, Vandana mentioned another sentence which i think is little bit more useful for the user: "Please click 'Login' in the upper right corner of the window"

I would add it to the current one so it becomes:
`You are not logged in. Please click 'Login' in the upper right corner of the window`

Reply:
Changed the message as what  it was in the ticket.